roles/partition-disk/handlers/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# handlers file for partition-disk
```

Comment:
If these files are gonn abe empty, I'd just delete them to reduce the noise

Reply:
In the Best-practices etherpad which I created and shared with @flatfender, I added this point to delete unused files created after we run `ansible-galaxy init`, but @flatfender  mentioned we maintain the dir_structure as it is. 

I think the idea of @flatfender was to have these as it is,  for the first/initial role, so that it be an example/reference for the directory structure we want to put in place.

Maybe moving forward we can remove it, ok will discuss with @flatfender  about it.

Reply:
Sounds good. I tend to think the docs on roles should suffice and the code repository should have as little noise as possible.

Reply:
Yes the idea was consistency around ansible-galaxy init, so that all roles have consistent structure and empty dirs don't hurt, and it makes it more galaxy friendly if we then commit back to galaxy. Once you delete them, then people tend to make things up later and we get inconsistency.